src/endless/EndlessUsbToolDlg.cpp

```diff
@@ -5381,6 +5381,14 @@ bool CEndlessUsbToolDlg::CanInstallToDrive(const CString & systemDriveLetter, co
 		}
 	}
 
+	// TODO: when this is called from the front page of the application, we do not know whether the boot.zip
+	// will contain eosldr stuff. If it does not, this check is still valid; if it does, we can skip all
+	// this (which is sort of the point). Ideas:
+	// - drop this up-front check, only perform it once the boot.zip is available (ie after a 10GB download)
+	// - perform this check after the to-be-installed image is selected, and hardcode the knowledge that
+	//   (master && date > 2017-02-...) || (>= eos3.2) => eosldr is available
```

Comment:
Definitely not adding to the manifest, this is a "one time" transition, and mixing/matching images with installers is really rare. Considering you can't actually get the tool to download an old image, the "omg I downloaded it but I can't install it" is probably impossible to arrive at after 3.2 is out - you'd only enter this state if you had an old local image, in which case option #2 and #1 are the same, and #1 is easiest to implement.